src/utils.ts

```diff
@@ -5,4 +5,5 @@ export { atomWithReducer } from './utils/atomWithReducer'
 export { atomFamily } from './utils/atomFamily'
 export { useSelector } from './utils/useSelector'
 export { useAtomCallback } from './utils/useAtomCallback'
+export { useAtomValue } from './utils/useAtomValue'
```

Comment:
This is just a minor thing, and we never had a strict rule so far.
But, how about moving this in L2, to be consistent with the order in docs?

Reply:
@dai-shi done.